website/pages/docs/plugins/sources/aws/event-based-sync.mdx

```diff
@@ -0,0 +1,92 @@
+import { Callout } from 'nextra-theme-docs';
+
+<Callout type="info">
+
+This feature is currently in a closed preview. Sign up for access using this [form](https://cloudquery.typeform.com/to/Dvdn6P2u)
+
+</Callout>
+
+# Event-Based Sync
+
+AWS CloudTrail enables users to get an audit log of events occurring within their account. By subscribing to a stream of AWS CloudTrail events in a Kinesis Data stream CloudQuery can trigger selective syncs to update just the singular resource that had a configuration change.
+
+
+## Supported Services and Events
+Service | Event 
+-|-|
+ec2.amazonaws.com | AttachInternetGateway
+ec2.amazonaws.com | AuthorizeSecurityGroupEgress
+ec2.amazonaws.com | AuthorizeSecurityGroupIngress
+ec2.amazonaws.com | CreateInternetGateway
+ec2.amazonaws.com | CreateSecurityGroup
+ec2.amazonaws.com | CreateSubnet
+ec2.amazonaws.com | CreateTags
+ec2.amazonaws.com | CreateVpc
+ec2.amazonaws.com | DeleteTags
+ec2.amazonaws.com | DetachInternetGateway
+ec2.amazonaws.com | ModifySubnetAttribute
+ec2.amazonaws.com | RevokeSecurityGroupEgress
+ec2.amazonaws.com | RevokeSecurityGroupIngress
+ec2.amazonaws.com | RunInstances
+iam.amazonaws.com | CreateGroup
+iam.amazonaws.com | CreateRole
+iam.amazonaws.com | CreateUser
+rds.amazonaws.com | CreateDBCluster
+rds.amazonaws.com | CreateDBInstance
+
+
+## Configuration
```

Comment:
Can we add a diagram of how this setup looks like?

For people not using the service yet it would be useful to know what's required on their end. Especially since the `streaming-deployment.yml` is private (maybe it shouldn't be).

If we could add a step by step guide how on to do it in the console that would be great (but non blocking for the PR). Maybe link to existing AWS docs/other guides regarding this topic

Reply:
Maybe we could create something similar to https://github.com/nexthink-oss/terraform-aws-cloudtrail-s3-to-kinesis/

Reply:
Yeah I think more information about this setup could definitely be added. Especially before this feature gets released publicly. Right now I am not that concerned because it is a closed preview

Reply:
> Right now I am not that concerned because it is a closed preview

The idea is to explain how the features works in a public to everyone, not only to the private beta users. If people don't know how this works, they might not register in the first place

Reply:
Fair. We can use the diagram that Michel has in the blog post